### PR TITLE
Remeasure amp-app-banner after built

### PR DESCRIPTION
**Pull request by muxin**
Fix https://github.com/ampproject/amphtml/issues/8358

### CODE REVIEW

**Comment by muxin** [comment]
@jridgewell PTAL

**Comment by jridgewell** [comment]
We should be able to remove the `setStyles` calls with this.

File: extensions/amp-app-banner/0.1/amp-app-banner.js [hunk 1]
```diff
@@ -126,11 +125,9 @@ export class AbstractAppBanner extends AMP.BaseElement {
       if (isDismissed) {
         this.hide_();
       } else {
-        setStyles(this.element, {
-          visibility: '',
-        });
         this.addDismissButton_();
         this.updateViewportPadding_();
+        this./*OK*/expand();
```
Comment (jridgewell) [inline extensions/amp-app-banner/0.1/amp-app-banner.js]: Please add a test.

Comment (muxin) [inline extensions/amp-app-banner/0.1/amp-app-banner.js]: Added test to verify `display` is not `none`.

**Comment by muxin** [comment]
Manually verified.

File: extensions/amp-app-banner/0.1/amp-app-banner.js (hunk 1 above)
Comment (jridgewell) [inline extensions/amp-app-banner/0.1/amp-app-banner.js]: It should be `''` at this point.

Comment (muxin) [inline extensions/amp-app-banner/0.1/amp-app-banner.js]: yeah, see https://github.com/ampproject/amphtml/pull/9388/files#diff-fee12a67efb59834e2b1db2548fab2a3R131

**Pull request merged**